src/main/java/com/freenow/apis/phrase/tasks/PhraseAppSyncTask.java

```diff
@@ -68,12 +70,27 @@ public PhraseAppSyncTask(final String authToken, final String projectId)
     */
     public PhraseAppSyncTask(final String authToken, final String projectId, final String scheme, final String host)
     {
-        this(authToken, projectId, Arrays.asList(DEFAULT_BRANCH), scheme, host);
+        this(authToken, projectId, null, singletonList(DEFAULT_BRANCH), scheme, host);
     }
 
-    public PhraseAppSyncTask(final String authToken, final String projectId, final List<String> branches, final String scheme, final String host)
+
+    /**
+     * Constructs a new PhraseAppSyncTask which downloads translation from a project tag.
+     * @param authToken Phrase auth token (optional)
+     * @param projectId ID of the project in Phrase
+     * @param tags List of comma separated tag names of the project in Phrase
+     * @param scheme URL scheme of the Phrase API, e.g. {@code https}
+     * @param host Host of the Phrase API, e.g. {@code api.phraseapp.com}
+     */
+    public PhraseAppSyncTask(final String authToken, final String projectId, final String tags, final String scheme, final String host)
+    {
+        this(authToken, projectId, tags, singletonList(DEFAULT_BRANCH), scheme, host);
+    }
+
+    public PhraseAppSyncTask(final String authToken, final String projectId, final String tags, final List<String> branches, final String scheme, final String host)
```

Comment:
Same idea as my comment on line 85: Remove `scheme` and `host` parameters in favor of the parameter `baseURL`.